builtin/reset.c

```diff
@@ -196,16 +196,44 @@ static int read_from_tree(const struct pathspec *pathspec,
 			  int intent_to_add)
 {
 	struct diff_options opt;
+	unsigned int i;
```

Comment:
nit on commit message: similarly here, I would leave back-ticks out of the top sentence:

```
reset: make --mixed sparse-aware
```

(Also: are we using "sparse-aware" versus "sparse aware" consistently? Let's make sure we are consistent within this series, at least.)

Reply:
Removed the backticks from this (and other commits). I've been using "sparse-aware" consistently in this series, as far as I can tell.

Reply:
I must have mis-remembered.